autoload/direnv.vim

```diff
@@ -86,3 +94,16 @@ function! direnv#export() abort
     call delete(l:tmp)
   endif
 endfunction
+
+let s:export_debounced = {'id': 0}
```

Comment:
If you change this to maybe do something like (only showing the `timers` branch of the `if`).

```VimL
let s:export_debounced = {'id': 0, 'counter': 0}

function! s:export_debounced.do()
  call timer_stop(self.id)
  if self.counter <= 5
    let self.counter = self.counter + 1
    let self.id = timer_start(s:direnv_interval, { -> let self.counter = 0 | direnv#export_core() })
  else
    let self.counter = 0
    let self.id = 0
    call direnv#export_core()
  endif
endif
```

If I’ve got the logic correct, every sixth call will result in a forced update regardless of anything else.

Reply:
I adopted it in d391b48.
Thank you very much.